src/Core/Common/CustomOperationToEachPixelInImage/Code.cxx

```diff
@@ -20,7 +20,7 @@
 #include "itkVariableLengthVector.h"
 #include "itkRigid2DTransform.h"
 #include "itkUnaryFunctorImageFilter.h"
-#include "vnl/vnl_math.h"
+#include "itkMath.h"
```

Comment:
> requirements as vnl_math_ has been recently replaced with a namespace of vnl_math::

Commit message needs changing. After reading it, I though changes would now use `vnl_math::`. Examples now use `itk::Math`, not `vnl_math::`.